contrib/platform/src/com/sun/jna/platform/win32/Advapi32.java

```diff
@@ -854,6 +855,28 @@ boolean GetTokenInformation(HANDLE tokenHandle,
     int RegOpenKeyEx(HKEY hKey, String lpSubKey, int ulOptions,
                      int samDesired, HKEYByReference phkResult);
 
+    /**
+     * The RegLoadAppKeyW function loads the specified registry hive
+     * as an application hive.
+     *
+     * @param lpFile
+     *            The name of the hive file.
+     *            If the file does not exist, an empty hive file is created
+     *            with the specified name.
+     * @param phkResult
+     *            Pointer to the handle for the root key of the loaded hive.
+     * @param samDesired
+     *            Access mask that specifies the desired access rights to the
+     * @param dwOptions
+     *            If this parameter is REG_PROCESS_APPKEY,
+     *            the hive cannot be loaded again while it is loaded by the caller.
+     *            This prevents access to this registry hive by another caller.
+     * @param Reserved
+     *            Reserved; must be zero.
+     */
+    int RegLoadAppKeyW(String lpFile, HKEYByReference phkResult,
```

Comment:
This should be `RegLoadAppKey`. The function name mapper for the w32 API (`com.sun.jna.win32.W32APIFunctionMapper`) takes care to get the `A` or `W` variant. This combined with `com.sun.jna.win32.W32APITypeMapper` make it possible to map both ascii and unicode variants together.